src/routes/accommodation.route.js

```diff
@@ -118,4 +118,69 @@ router.get('/:accomodationId', verifyToken.headerToken, verifyUser, Accommodatio
  *            type: string
  */
 router.post('/booking', Validate.bookingValidation(), isValid, verifyToken.headerToken, AccomodationMiddleware.validateDates, AccomodationMiddleware.checkBookingFacilitiesAvailability, Accommodation.bookAccommodation);
+router.post('/rate/:accomodationId', verifyToken.headerToken, verifyUser, AccomodationMiddleware.checkValidAccomodationRates, AccomodationMiddleware.checkUserAccomodationCorrespondance);
+/**
+ * @swagger
+ *
+ * /accommodations/rate/{accomodationId}:
+ *    post:
+ *      summary: user should be to rate an accommodation
```

Comment:
@mujohn26 I think what you've forgotten is that a user can update their ratings. There's a need for that route too. Additionally, we need to display ratings a user has given, given the accommodationID and the userID. There's a need for that route too.

Reply:
Done